exchanges/stream/websocket.go

```diff
@@ -270,6 +270,8 @@ func (w *Websocket) Connect() error {
 			w.exchangeName)
 	}
 
+	w.subscriptions = subscriptionMap{}
```

Comment:
```suggestion
        w.subscriptionMutex.Lock()
	w.subscriptions = subscriptionMap{}
	w.subscriptionMutex.Unlock()
```
Arguable whether its needed, but its used anywhere else it gets touched

Reply:
Agreed.

Fixed c71460f8